pre-commit.md

```diff
@@ -1,18 +1,14 @@
 # Setting up Pre-Commit
 
-When creating a new Python code repository, the preferred approach is to create a new branch from the template repo
-(see [new repository template](new_repository.md#Repository Template)). Where that is not possible e.g. adding
-pre-commit to an existing repository, please add these configuration files into the root folder.
+When creating a new Python code repository, the preferred approach is to create a new branch from the template repo (see [new repository template](new_repository.md#Repository Template)). Where that is not possible e.g. adding pre-commit to an existing repository, please add these configuration files into the root folder.
 
-(you can skip the last 3 lines if your project is not going to contain any Python code):
+(you can skip the last 3 lines if your project is not going to contain any python code):
```

Comment:
```suggestion
(you can skip the last line if your project is not going to contain any python code):
```
Just the last line now that pylint/flake8 are scrubbed?

Reply:
Deleted this files because I removed all the template files from here, and it should be clear enough on the _setting up a new repo_ doc.